docs/blog/posts/argocd-keptn-health.md

```diff
@@ -0,0 +1,163 @@
+---
+date: 2024-05-15
+authors: [odubajDT]
+description: >
+  In this blog post we will present how Keptn can show application health in integration with ArgoCD.
+categories:
+  - ArgoCD
+  - Application health
+comments: true
+---
+
+# Application health checks with Keptn using ArgoCD
+
+In this blog post we will present a planned Keptn and
+[ArgoCD](https://argo-cd.readthedocs.io/en/stable/) integration to execute
+advanced application health checks using Keptn, where the application itself is deployed
+by ArgoCD and display the Keptn application health status via ArgoCD UI.
+
+Keptn provides an effective way to perform application health checks using the
+pre- or post-deployment tasks and evaluations.
+Compared to ArgoCD application
+health checks, which are evaluating if the application is successfully deployed
+and the workloads are running on the cluster, they do not show if the microservices
+of a single application are able to communicate and cooperate with each other.
```

Comment:
this sounds like Keptn specifically checks for the communication between the services. Can you rephrase this a bit to highlight that these kind of checks are just an example, and any kind of check can be done? maybe something like 
```suggestion
and the workloads are running on the cluster, they do not show if the microservices
of a single application are actually working as expected.
For example, it could be the case that the individual services are up and running, but due to a
networking misconfiguration they cannot communicate with each other.
```